src/tests/Button.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Button from '../components/Button';
+
+describe('testing Button', () => {
```

Comment:
- [ ] Good job so far by adding the previous reviewer's requested changes but I think while you were adding the user interaction tests for Quote some errors are coming. Please make sure to fix all the errors in your tests and make them pass without errors like before 👍 

![error](https://user-images.githubusercontent.com/6615394/229107710-1baa363e-58f2-422d-b873-00f48fdc8021.PNG)